package.json

```diff
@@ -17,21 +17,22 @@
     "sax": ">=0.6",
     "selectn": "^0.9.6",
     "strip-bom": "~0.3.1",
-    "xml-crypto": "~0.8.0"
+    "xml-crypto": "~0.8.0",
+    "ursa": "0.8.5 || >=0.9.4"
```

Comment:
`ursa` was made an `optionalDependency` as most of the users had problems with the library. please revert the change in the `dependencies` list as it is listed under `optionalDependencies` which should be sufficient.